src/applications/education-letters/actions/index.js

```diff
@@ -1,29 +1,49 @@
 import { apiRequest } from 'platform/utilities/api';
 import environment from 'platform/utilities/environment';
 
-export const CLAIM_STATUS_ENDPOINT = `${
+export const MEB_CLAIM_STATUS_ENDPOINT = `${
   environment.API_URL
 }/meb_api/v0/claim_status`;
+export const TOE_CLAIM_STATUS_ENDPOINT = `${
+  environment.API_URL
+}/meb_api/v0/forms_claim_status`;
+
+export const CLAIM_STATUSES = [];
 
 export const FETCH_CLAIM_STATUS = 'FETCH_CLAIM_STATUS';
 export const FETCH_CLAIM_STATUS_SUCCESS = 'FETCH_CLAIM_STATUS_SUCCESS';
 export const FETCH_CLAIM_STATUS_FAILED = 'FETCH_CLAIM_STATUS_FAILED';
 
+const fetchAPIs = (endpoints, dispatch) =>
+  endpoints
+    .map(endpoint =>
+      apiRequest(endpoint)
+        .then(response => {
+          dispatch({
+            type: FETCH_CLAIM_STATUS_SUCCESS,
+            response,
+          });
+        })
+        .catch(errors => {
+          dispatch({
+            type: FETCH_CLAIM_STATUS_FAILED,
+            errors,
+          });
+        }),
+    )
+    .reduce((currentDate, nextDate) => {
```

Comment:
It might be a little cleaner to have the reducing in in reducers.js, but not the end of the world if you like it here.

Reply:
Thanks, will update it soon. Converting this PR to draft for the moment.